book/P1C6_survtsk.qmd

```diff
@@ -10,119 +10,270 @@ abstract: "This chapter introduces the different survival problems and formalise
+
+A potential confusion that should be avoided is conflating this risk prediction with a prediction of absolute risk.
+Risk predictions are specifically *relative risks*, which means the risks are only comparable to other observations within the same sample.
+For example, given three subjects, $\{i,j,k\}$, a risk prediction may be $\{0.5, 10, 0.1\}$ respectively.
+From these predictions, two primary types of conclusion can be drawn.
+
+1) Conclusions comparing subjects:
 
-* Conclusions comparing subjects. For example, $i$ is at the least risk; the risk of $j$ is only slightly higher than that of $i$ but the risk of $k$ is considerably higher than $j$; the corresponding ranks for $i,j,k,$ are $1,2,3$;
-* Conclusions comparing risk groups. For example, thresholding the risks at $1.0$ means that $i$ and $j$ are in a low-risk group whilst $k$ is in a high-risk group.
+* The corresponding ranks for $i,j,k$ are $2,3,1$;
+* $k$ is at the least risk and $j$ is at the highest risk;
+* The risk of $i$ is slightly higher than that of $k$ but $j$'s risk is considerably higher than both the others.
+
+2) Conclusions comparing risk groups:
+
+* Thresholding risks at $0.4$ means $k$ is at a low-risk but $i$ and $j$ are high-risk.
+* Thresholding risks at $1.0$ means $i$ and $k$ are low-risk but $j$ is high-risk.
 
 Whilst many important conclusions can be drawn from these predictions, the values themselves have no meaning when not compared to other individuals.
-Interpretation of these rankings depends on the model class (for example, PH and AFT models have opposite interpretations, @sec-models-classical) and its parametrization or implementation in specific software.
-For some higher ranking implies higher risk whereas others may assume that higher ranking implies lower risk.
-In this book, a higher ranking will always imply a higher risk of event (as in the example above).
+Similarly, the values have no meanings across research, even if observation $k$ is at low-risk according to this sample, it may be high risk compared to another.
+
+### Distributions and risks
+
+In general it is not possible to easily convert a risk prediction to a distribution.
+However, this may be possible if the risk prediction corresponds to a particular model form.
+This is discussed further in @sec-survtsk-PI.
 
-Predicting rankings is the primary form of the *survival ranking task*, defined by predicting a continuous value, $g: \calX \rightarrow \calR$ where $\calR \subseteq \Reals$.
+More common is transformation from a distribution to a risk.
+Theoretically the simplest way to do so would be to take the mean or median of the distribution, however as will be discussed in detail in @sec-survtsk-time this is practically difficult.
+Instead, a stable approach is to use the 'ensemble' or 'expected mortality' to create a measure of risk [@Ishwaran2008].
+The expected mortality is defined by
+
+$$
+\sum_t -\log(\hatS_i(t)) = \sum_t \hatH_i(t)
```

Comment:
so is the advantage of this vs. just using S_i(t) that it also works for crossing S_i(t)'s? Otherwise because S_i(t) and H_i(t) are monotonous the ranking would be the same?